MyOwnORM/MyOwnORM/Context/CustomDbContext.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbContext : IDisposable
+    {
+        private readonly string _connectionString = string.Empty;
+        private bool disposed = false;
+        public CustomDbContext()
+        {
+
+        }
+
+        public CustomDbContext(string connectionString)
+        {
+            _connectionString = connectionString;
+        }
+
+        public string ConnectionString
+        {
+            get { return _connectionString; }
+        }
+
+        protected virtual void OnConfiguring(CustomDbContextOptionsBuilder optionsBuilder) 
+        {
+
+        }
+
+        public void EnsureCreated()
```

Comment:
you don't need this. because you have not functionality to create db. You can left it but do something here(seed data as example). If not, delete method

Reply:
Done

Reply:
Method actually exists.

Reply:
Fixed